app/models/organization.rb

```diff
@@ -0,0 +1,6 @@
+class Organization < ApplicationRecord
+  # Associations
+  has_many :tradings
+  # Validations
+  validates :name, presence: true, uniqueness: true, length: { maximum: 30 }
```

Comment:
IMHO, we can remove the comments, as its not that necessary.

Reply:
removed vaiya, thank you